### PR TITLE
Update jobs, team, and processes

### PR DESCRIPTION
**Pull request by mbabic**

### CODE REVIEW

File: content/en/about/day-in-life-dev.md [hunk 1]
```diff
@@ -6,95 +6,111 @@ position: 7
 category: 'About Us'
```
Comment (mbabic) [inline content/en/about/day-in-life-dev.md:6]: Ack, I was too aggressive with the auto-formatter.

Will point out non-formatting changes and where I'd like feedback.

File: content/en/about/day-in-life-dev.md [hunk 2]
```diff
@@ -6,95 +6,111 @@ position: 7
...
+
 ### \#daily-gsd
 
-We use [Slack](https://slack.com/) to stay connected with the rest of the team. 
+We use [Slack](https://slack.com/) to stay connected with the rest of the team.
 
-The `#daily-gsd` channel lets us keep track of what our teammates are working on and identify blockers as well as potential conflicts.
-Each one of us starts the day by taking a look at his/her work queue, what (s)he achieved yesterday, and what (s)he plans to complete today, 
-and writes a 2-3-liner in `#daily-gsd` to let the rest of the team know about the current priorities.
+The `#daily-gsd` channel lets us keep track of what our teammates are working on and identify blockers as well as
```
Comment (mbabic) [inline content/en/about/day-in-life-dev.md:18]: Update: removed gendered language.

File: content/en/about/day-in-life-dev.md [hunk 3]
```diff
@@ -6,95 +6,111 @@ position: 7
...
+potential conflicts.
+Each of us starts the day by taking a look at our work queue, what we achieved yesterday, and what we plan to complete
+today, then we write a couple sentences in `#daily-gsd` to let the rest of the team know about the current priorities.
 
 ### \#auto-techops
```
Comment (mbabic) [inline content/en/about/day-in-life-dev.md:23]: Update: removed deprecate PS rotation stuff, folded it into on call.

File: content/en/about/day-in-life-dev.md [hunk 4]
```diff
@@ -6,95 +6,111 @@ position: 7
...
+The on call will also monitor the `#partner-success` and `#helpdesk` channels to prevent partners and the business from
+beingn blocked on technical issues.
 
-There is also a 30-minute meeting on Monday with the sales team to proritize the work and get an idea about the upcoming, sales/partner-driven
-requests.
+There is also a 30-minute meeting on Monday with the sales team to proritize the work and get an idea about the
+upcoming sales/partner-driven requests.
 
-### Crucible reviews
+### Code Reviews
```
Comment (mbabic) [inline content/en/about/day-in-life-dev.md:34]: Update: got rid of mentions of Crucible.

This section feels a bit out of date, Don't think anyone does code review meetings anymore, but left it as is.

File: content/en/about/day-in-life-dev.md [hunk 5]
```diff
@@ -6,95 +6,111 @@ position: 7
...
 
-### Daily standup (2PM EST)
+### Daily standup (14:30 EST)
 
-We meet on Google Meet every day at 1:30 PM EST for a quick, 15-minute conversation to quickly synchronize and address any blockers. 
+We meet on Google Meet every day at 14:30 EST for a quick, 15-minute conversation to quickly synchronize and address
+any blockers.
 
-### Weekly dev meeting
+### Dev Weekly
```
Comment (mbabic) [inline content/en/about/day-in-life-dev.md:60]: Left this as is mostly even though we don't do it on Tuesdays anymore, figure this is true enough while we rejigger our meetings.

File: content/en/about/day-in-life-dev.md [hunk 6]
```diff
@@ -6,95 +6,111 @@ position: 7
...
-- Improving the performance of an existing spark report by reducing the amount of shuffling it has to do.
-- Writing a new spark report to generate analytics on data that we have ingested.
-- Creating a frontend component using d3.js to display the aforementioned analytics.
-- Setting up new formats to ingest data that a new partner is sending to us.
-- Extracting reusable helpers into the narrative-commons to remove duplication across the jobs.
-
+Outside of the on call rotation, the bulk of the work involves implementation and coding. This covers everything
+from improving an existing feature to setting up an entirely new service to implement a new feature.
 
 ### Deployment and Ops
```
Comment (mbabic) [inline content/en/about/day-in-life-dev.md:72]: This and design reviews are untouched.

File: content/en/about/day-in-life-dev.md [hunk 7]
```diff
@@ -6,95 +6,111 @@ position: 7
...
+### Dev Weekly
 
-We gather on Google Meet once a week (Tuesday at 1:30 EST) for a 1-hour discussion with the dev team. This is the opportunity to raise awareness about 
-any issues, suggest improvements to the process, seek feedback on ideas, etc.
+We gather on Google Meet once a week (Tuesday at 14:30 EST) for a 1 hour discussion with the dev team. This is the
+opportunity to raise awareness about any issues, suggest improvements to the process, seek feedback on ideas, etc.
 
 ## Development
 
 ### Code
```
Comment (mbabic) [inline content/en/about/day-in-life-dev.md:67]: Update: removed example projects because they felt a bit stale. Updated the example projects in the job posting, but those feel bigger/aspirational and the ones we have here should be more grounded and day-to-day, but ran out of energy trying to think of what this should be.

If someone wants to take a crack at this please do 🙇

File: content/en/about/day-in-life-dev.md [hunk 8]
```diff
@@ -6,95 +6,111 @@ position: 7
...
 
-Writing the design document sometimes involves prototyping and experimenting with technologies, depending on the nature of the work.
+Writing the design document sometimes involves prototyping and experimenting with technologies, depending on the nature
+of the work.
 
-## A typical calendar
+## A Typical Calendar
 
-We attempt to avoid [swiss cheesing our calendars](https://thinkingthrough.substack.com/p/stop-swiss-cheesing-your-calendar) 
+We attempt to
```
Comment (mbabic) [inline content/en/about/day-in-life-dev.md:103]: Left this section as is while we re-jigger our meetings. Hopefully not too untrue.

File: content/en/about/technical-stack.md [hunk 9]
```diff
@@ -5,38 +5,45 @@ position: 8
...
+functional programming and make use of libraries like `cats`, `cats-effect`, `http4s`, `doobie`, but we keep a
+pragmatic stance on purity and have to play nice with the Hadoop and Spark ecosystems.
 
 Here are some highlights of the major backend components:
-- `open-api`: an `http4s` API running on `Fargate` to submit (buy, sell) orders to the marketplace
-- `ingestion`: spark jobs that ingest supplier data and store it in the data lake and puddles (Apache Iceberg).
-- `transaction`: spark jobs that transact the data between the buyers and sellers
-- `delivery`: spark jobs triggered by `AWS Step Functions` that deliver the transacted data to a format suitable for the buyers' consumption
+
+- `api`: an `http4s` API running on Fargate that is used by any person or process interacting with our platform (e.g.
```
Comment (mbabic) [inline content/en/about/technical-stack.md:26]: Tried to update this a bit but don't feel like I did a great job (e.g. `apps` is too wordy). If you can think of some better, more communicative bullet points lmk.

File: content/en/open-positions/backend-engineer.md [hunk 10]
```diff
@@ -1,45 +1,53 @@
...
-  For instance, we are building Iceberg-backed data puddles, schema/validation tools, making the transaction process more generic, etc.
-- Improve the monitoring and reliability of our systems with the ever-increasing amount of data being handled (over 500GB per day)
-- Extract abstractions, modularize the code and improve the tests.
-- Enhance the API we expose to partners and increasingly rely on it for internal use, in order to "eat our own dogfood".
-
+We have built a marketplace that allows buyers and sellers to transact any kind of data.
+
+Here are the kinds of projects you will likely work on:
+
+- Add capabilities to our query execution and data classification engines to make it possible to fulfill any buyer
```
Comment (mbabic) [inline content/en/open-positions/backend-engineer.md:14]: This needs another set of eyes. I think I focused too much on the the query part of the product and not enough on the marketplace part, taking any and all feedback (e.g. suggestions for what too add, what to remove, etc.)

File: content/en/process/dev-hiring-process.md [hunk 11]
```diff
@@ -5,18 +5,21 @@ position: 9
 category: Process
 ---
 
-The goal of the hiring process is to give the opportunity to both sides to assess the fit. It is as much an opportunity for the candidate to decide whether they want
```
Comment (mbabic) [inline content/en/process/dev-hiring-process.md:8]: Just reformatting and cosmetic changes to the text.

File: content/en/sections/common-requirements.md [hunk 12]
```diff
@@ -1,9 +1,8 @@
-- Have strong experience using a version management system and continuous integration (CI) development process. We use `Git`/`Github` for version management and `Gitlab` for our `CI` pipelines.
```
Comment (mbabic) [inline content/en/sections/common-requirements.md:1]: Updates:

- got rid of version management from requirements and CI, I haven't seen a resume where someone doesn't mention some kind of CI + Github / Gitlab.
- Added "Drive success in a flat organization with minimal process, interfacing with technical and non-technical team members as necessary."

File: content/en/sections/mission.md [hunk 13]
```diff
@@ -1,4 +1,4 @@
-We are building a data streaming marketplace that makes it easy to buy, sell and win.
+We are building a data streaming marketplace that makes it easy to buy, sell, and win.
```
Comment (mbabic) [inline content/en/sections/mission.md:1]: We decided we were an oxford comma company awhile ago. 🤷

File: content/en/open-positions/backend-engineer.md [hunk 14]
```diff
@@ -1,45 +1,53 @@
...
-to make it possible to transact any other kinds of data.
-
-As a result, here are the kinds of projects you will likely work on in the foreseable future:
-- Make our core systems generic/agnostic to the actual data types that are being ingested, transacted and delivered. 
-  For instance, we are building Iceberg-backed data puddles, schema/validation tools, making the transaction process more generic, etc.
-- Improve the monitoring and reliability of our systems with the ever-increasing amount of data being handled (over 500GB per day)
-- Extract abstractions, modularize the code and improve the tests.
-- Enhance the API we expose to partners and increasingly rely on it for internal use, in order to "eat our own dogfood".
-
+We have built a marketplace that allows buyers and sellers to transact any kind of data.
```
Comment (mbabic) [inline content/en/open-positions/backend-engineer.md]: Hm just realizing this sentence is weird on its own. Can't think of a good blurb here. Maybe the old preamble of

> We currently have a marketplace that connects buyers and sellers of adtech data, and we are building on this success to make it possible to transact any other kinds of data.

Is still accurate.

File: content/en/about/team.md [hunk 15]
```diff
@@ -5,14 +5,14 @@ position: 6
 category: 'About Us'
 ---
 
-## The Team (Remote US and Canada)
+## The Team
```
Comment (MasseGuillaume) [inline content/en/about/team.md:8]: Do we have a mention about Skuad and the fact we can hire anywhere in the world? I think this is a huge plus. They do our taxes, we do not need to worry about visa/immigration.

File: content/en/about/day-in-life-dev.md [hunk 16]
```diff
@@ -6,95 +6,111 @@ position: 7
...
-things that need to be fixed, failed clusters, etc.
-
-### \#partner-success and \#helpdesk
+The person who is on call will look at the various warnings and alerts in the `#auto-techops` channel in order to
+identify things that need to be fixed, failed clusters, etc.
 
-The person who is on Partner Success (PS) rotation will monitor the `#partner-success` and `#helpdesk` channels, as well as the Partner 
-Success queue in [Clubhouse](http://clubhouse.io/) in order to determine the next priorities.
+The on call will also monitor the `#partner-success` and `#helpdesk` channels to prevent partners and the business from
+beingn blocked on technical issues.
```
Comment (samidalouche) [inline content/en/about/day-in-life-dev.md]: being

Comment (mbabic) [reply content/en/about/day-in-life-dev.md]: Thanks Sami. Fixed in [199a36c](https://github.com/narrative-io/narrative-jobs/pull/4/commits/199a36c6a0d366b9c3b4c47f6d7b28018978c1a5)

File: content/en/about/team.md [hunk 17]
```diff
@@ -5,14 +5,14 @@ position: 6
...
+features from design to launch.
 
 We are building the team with a remote-first mindset, and as a result every team member is expected to have an ability
 to synthesize business requirements, distill the domain, contribute to high-level design documents, efficiently
 communicate asynchronously, and more generally work autonomously toward a shared vision.
 
-While the company's headquarter is in NYC, the development team is currently made of engineers working from the US (California),
-Canada (Alberta, British Columbia, and Québec), and Germany.
+While the company's headquarters are in NYC, the development team is currently made of engineers working from the US
+(California and Florida), Canada (Alberta, British Columbia, and Québec), Germany, and Nigera.
```
Comment (mbabic) [inline content/en/about/team.md:18]: @MasseGuillaume added the missing locations. Thanks for pointing it out. :)

File: content/en/about/team.md [hunk 18]
```diff
@@ -5,14 +5,14 @@ position: 6
...
+features from design to launch.
 
 We are building the team with a remote-first mindset, and as a result every team member is expected to have an ability
 to synthesize business requirements, distill the domain, contribute to high-level design documents, efficiently
 communicate asynchronously, and more generally work autonomously toward a shared vision.
 
-While the company's headquarter is in NYC, the development team is currently made of engineers working from the US (California),
-Canada (Alberta, British Columbia, and Québec), and Germany.
+While the company's headquarters are in NYC, the development team is currently made of engineers working from the US
+(California), Canada (Alberta, British Columbia, and Québec), and Germany.
```
Comment (MasseGuillaume) [inline content/en/about/team.md]: Nigeria! 🇳🇬 
US (Florida)

I made an interactive map: https://jsfiddle.net/jLbeo81t/1/

File: content/en/index.md [hunk 19]
```diff
@@ -11,12 +11,12 @@ category: ''
 ## Open Positions
 
 Narrative I/O is hiring:
-- [Scala/AWS/Spark Backend Engineers (Remote US and Canada)](/open-positions/backend-engineer)
-- [Web Software Engineer (Remote US and Canada)](/open-positions/web-software-engineer)
 
-We are currently planning to hire several engineers throughout 2021. If you are interested to explore opportunities 
-but cannot make an immediate move, do not hesitate to reach out, we are flexible. 
+- [Senior Backend Engineer (Remote US and Canada)](/open-positions/backend-engineer)
+- [Web Software Engineer (Remote US and Canada)](/open-positions/web-software-engineer)
```
Comment (MasseGuillaume) [inline content/en/index.md:16]: ah it's limited to us and canada only ?

File: content/en/process/dev-hiring-process.md [hunk 20]
```diff
@@ -28,60 +31,79 @@ The interview consists of three stages:
...
-- +/- 3 hours EST. The further from EST, the stronger the rest of the attributes need to be.
-- Has worked remotely before
-- Has worked for a small (< 15 developers, < 50 employee) company
-- Has worked with specific technology in our stack -- Scala, AWS, Big Data, etc
-- Good written communicator, initial communication is clear and understandable
-- (Bonus): initial cover email is not a s/$COMPANY/Narrative template. Did this person even read the job req?
+- +/- 3 hours EST. The further from EST, the stronger the rest of the attributes need to be.
+- Has worked remotely before.
+- Has worked for a small (< 15 developers, < 50 employee) company.
+- Has worked with specific technology in our stack: Scala, AWS, "Big Data", etc.
```
Comment (MasseGuillaume) [inline content/en/process/dev-hiring-process.md]: Spark (better be specific :))

**Review by MasseGuillaume — APPROVED** [review]

File: content/en/about/team.md (hunk 15 above)
Comment (mbabic) [reply content/en/about/team.md:8]: I had that in there at one point but business's pref is to target US/Canada for time zone overlap + simplifying the onboarding process, so for now so I removed it. If we open up remote from anywhere again I'll make sure to add it here, because yeah just saying "you can work from anywhere as long as our EOR supports you" would be important to mention.

File: content/en/process/dev-hiring-process.md (hunk 20 above)
Comment (mbabic) [reply content/en/process/dev-hiring-process.md]: Good call. Fixed in bbc8002e2937c757375680c2a1d81c4ff5ffcbc7

File: content/en/open-positions/backend-engineer.md (hunk 14 above)
Comment (mbabic) [reply content/en/open-positions/backend-engineer.md]: Reverted to the old preamble for now.
